controllers/containerimagebuild_resources.go

```diff
@@ -378,7 +378,16 @@ func (r *ContainerImageBuildReconciler) createJobForBuild(ctx context.Context, c
 		imagePullSecrets = append(imagePullSecrets, corev1.LocalObjectReference{Name: r.JobConfig.ImagePullSecret})
 	}
 
-	// construct final job object
+	var tolerations []corev1.Toleration
+	if r.JobConfig.TolerationKey != "" {
+		var toleration = corev1.Toleration{
+			Key: r.JobConfig.TolerationKey,
+			Operator: "Exists",
+		}
+		tolerations = append(tolerations, toleration)
```

Comment:
```suggestion
		tolerations = append(tolerations,  corev1.Toleration{
			Key: r.JobConfig.TolerationKey,
			Operator: "Exists",
		})
```

or

```golang
		toleration := corev1.Toleration{
```

using var for assignment isn't encouraged